applications/asset_tracker/src/main.c

```diff
@@ -348,6 +356,41 @@ static void cloud_cmd_handler(struct cloud_command *cmd)
 	/* Command handling goes here. */
 }
 
+#if CONFIG_BATTERY_MONITOR
+static void battery_monitor_check(struct k_work *work)
+{
+	static u8_t bat_charge;
+	static bool bat_level_acceptable = true;
+
+	battery_monitor_read(&bat_charge);
+
+	if (bat_charge < CONFIG_BATTERY_MONITOR_THRESHOLD &&
+	    bat_level_acceptable == true) {
+		ui_led_set_color(CONFIG_BATTERY_MONITOR_RED_VALUE,
```

Comment:
I find it confusing to have `set_color` function accept not only the color, but also the on/off period.

I saw that there was a `UI_BATTERY_LEVEL_LOW` led pattern specified in this PR. Why don't we use it here instead? You could:
1) When the battery level drops below the threshold, store the current pattern and apply the `UI_BATTERY_LEVEL_LOW`.
2) Once battery is connected/charged, restore the previously stored pattern, as suggested by @jhn-nordic.

Of course, you would need to handle situations, when the pattern is being changed while the battery level is low. A simple wrapper over `ui_led_set_patttern` should handle that though.